### PR TITLE
fix(stock_controller): precision based on `debit_in_account_currency`

### PR DESCRIPTION
**Pull request by iRaySpace**
**Overview**
This is a new PR to existing PR:
https://github.com/frappe/erpnext/pull/22860

This is a logic already added to the existing branch. However, `version-11` branch don't have this code yet.
You may refer it here: https://github.com/frappe/erpnext/blob/develop/erpnext/controllers/stock_controller.py#L74

### CODE REVIEW

**Comment by f-9t9it** [comment]
@deepeshgarg007 
Please look into this.

**Comment by Muzzy73** [comment]
@rohitwaghchaure Kindly look into this. This effects countries with 3 decimal digit currencies.

File: erpnext/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from erpnext.hooks import regional_overrides
 from frappe.utils import getdate
 
-__version__ = '11.1.68'
+__version__ = '11.1.77'
```
Comment (rohitwaghchaure) [inline erpnext/__init__.py:8]: why changes in this file? can you do the rebase

**Comment by rohitwaghchaure** [comment]
25 commits to make changes in one file, it seems that the issue with rebase. Closing this PR, can you please open new PR with single commit

**Pull request closed without merging**